src/OpenGL/Silk.NET.OpenGL/GL.cs

```diff
@@ -233,63 +615,170 @@ public void VertexAttrib4(uint index, Vector4 v)
+        ///      The returned values.
+        /// </param>
+        /// <remarks>
+        ///  <para>
+        ///      A list of valid symbolic constants (parameter names) can be found in the documentation at the provided link.
+        ///  </para>
+        ///  <para>
+        ///      <see href="https://www.khronos.org/registry/OpenGL-Refpages/gl2.1/xhtml/glGet.xml">OpenGL Documentation</see>
+        ///  </para>
+        /// </remarks>
         public void GetFloat(GLEnum pname, out Matrix4x4 matrix)
         {
             unsafe
             {
                 fixed (Matrix4x4* ptr = &matrix)
                 {
-                    GetFloat(pname, (float*)ptr);
+                    GetFloat(pname, (float*) ptr);
                 }
             }
         }
 
-        public void Viewport(Size size)
-        {
-            Viewport(0, 0, (uint)size.Width, (uint)size.Height);
-        }
-
-        public void Viewport(Point location, Size size)
-        {
-            Viewport(location.X, location.Y, (uint)size.Width, (uint)size.Height);
-        }
-
-        public void Viewport(Rectangle rectangle)
-        {
-            Viewport(rectangle.X, rectangle.Y, (uint)rectangle.Width, (uint)rectangle.Height);
-        }
+        /// <summary>
+        ///     Set the viewport for the OpenGL context.
+        /// </summary>
+        /// <param name="size">
+        ///     Specifies the width and height of the viewport.
+        /// </param>
+        /// <remarks>
+        ///  <para>
+        ///     The location of the viewport will default to 0,0.
+        ///  </para>
+        ///  <para>
+        ///      When an OpenGL context is first attached to a window, width and height are set to the dimensions of that window.
+        ///  </para>
+        ///  <para>
+        ///     <see href="https://www.khronos.org/registry/OpenGL-Refpages/gl4/html/glViewport.xhtml">OpenGL Documentation</see>
+        ///  </para>
+        /// </remarks>
+        public void Viewport(Size size) { Viewport(0, 0, (uint) size.Width, (uint) size.Height); }
+
+        /// <summary>
+        ///      Set the viewport for the OpenGL context.
+        /// </summary>
+        /// <param name="location">
+        ///      Specifies the lower left corner of the viewport rectangle, in pixels.
+        /// </param>
+        /// <param name="size">
+        /// Specifies the width and height of the viewport.
+        /// </param>
+        /// <remarks>
+        ///  <para>
+        ///      When an OpenGL context is first attached to a window, width and height are set to the dimensions of that window.
+        ///  </para>
+        ///  <para>
+        ///      <see href="https://www.khronos.org/registry/OpenGL-Refpages/gl4/html/glViewport.xhtml">OpenGL Documentation</see>
+        ///  </para>
+        /// </remarks>
+        public void Viewport(Point location, Size size) { Viewport(location.X, location.Y, (uint) size.Width, (uint) size.Height); }
```

Comment:
This change goes against the code style for the long line limit of 120 characters and also should be an expression-style body. You can solve this using:
```suggestion
        public void Viewport(Point location, Size size)
            => Viewport(location.X, location.Y, (uint) size.Width, (uint) size.Height);
```